HelpAssistHer/help-assist-her
# Add prettier

## Description
- In order to have consistent style and not have the need to have discussions around style, we are introducing Prettier, which takes care of code formatting for us. See the CONTRIBUTING doc for more info.
- In addition, I also moved the local setup instructions to a SETUP readme and updated the Contributing doc with more info on PRs and Prettier

## Test Plan
- Make a change to a file
- Stage it `git add file-name`
- Run `yarn precommit`
- Verify that Prettier runs on the staged file

## Merge Plan
This PR will not be a big deal to merge. However, once it's merged, Prettier will run on all commits. This means that when someone updates a file, it will show a lot of changes, most of which will be formatting changes, because our current code does _not_ follow this style so well. 😱 This will make PR reviews a BEAR! So here is my plan to minimize the blast radius:
1. Get this PR reviewed/approved
1. Get all other open PRs reviewed/approved/merged into master
1. Merge this PR
1. Before opening up any other PRs, I will go through and run Prettier on all files. I'll commit the files directly to master since there won't be any functionality change.
1. When new PRs are opened, we will only see the changes the developer made and they will be prettily formatted 😄
